app/presenters/concerns/decisions_api_data.rb

```diff
@@ -37,7 +37,7 @@ def rejection
   end
 
   def offer
-    return nil if application_choice.offer.nil?
+    return nil if ApplicationStateChange::OFFERED_STATES.exclude?(application_choice.status.to_sym)
```

Comment:
I had a look through the states, and realised that withdrawn isn't there, if an application is withdrawn after it reaches pending conditions, we would lose the offer appearing in the response. I think we'll have to rely on the offer object being there unfortunately and not the states after all :(

Reply:
Reverted to using the offer object